release-notes/opensearch-security.release-notes-1.3.6.0.md

```diff
@@ -0,0 +1,10 @@
+## 2022-10-03 Version 1.3.6.0
+
+### Bug fixes
+* Update Kafka Client to 3.0.2 ([#2129](https://github.com/opensearch-project/security/pull/2129))
+* Scope updateVersion to only build.gradle ([#2121](https://github.com/opensearch-project/security/pull/2121))
+* Staging for version increment automation ([#2029](https://github.com/opensearch-project/security/pull/2029))
```

Comment:
This could probably go in Maintenance too

Reply:
Hey @vinayak15 we have some discretion when it comes to categorizing the issues, but generally speaking security patches will be categorized into Maintenance since that is ongoing maintenance we perform on the plugin and is not a bug in the security plugin itself.

The second one is a bugfix for version increment automation where it wasn't scoped correctly.

The third one could be argued as maintenance or enhancement IMO. I would moreso argue for maintenance because its not an enhancement to the security plugin, rather it helps us automate the version increment process between releases. 

You can read more about plugin release notes here: https://github.com/opensearch-project/opensearch-plugins/blob/main/RELEASE_NOTES.md